content/applications/sales/crm/performance/google_spreadsheets.rst

```diff
@@ -48,7 +48,7 @@ Link a Spreadsheet with Odoo
 Applications
 ============
 
-You have 2 different formulas of using Google Spreadsheet in Odoo: retrieve data and retrieve
+You have 2 different formulas available when using Google Spreadsheet in Odoo: retrieve data and retrieve
```

Comment:
The grammar's good here too.

Let's also add _italics_ around `retrieve data` and `retrieve grouped sums` to indicate that they are specific formulas and not just generic copy on the page.